src/crab_verifier.cpp

```
@@ -62,15 +62,25 @@ Invariants analyze(const Program& prog, const StringInvariant& entry_invariant)
     return analyze(prog, EbpfDomain::from_constraints(entry_invariant.value(), thread_local_options.setup_constraints));
 }
 
+static std::optional<VerificationError> check_loop_bound(const Program& prog, const Label& label,
+                                                         const EbpfDomain& pre) {
+    if (std::holds_alternative<IncrementLoopCounter>(prog.instruction_at(label))) {
+        const auto assertions = prog.assertions_at(label);
+        if (assertions.size() != 1) {
+            CRAB_ERROR("Expected exactly 1 assertion for IncrementLoopCounter");
```

Clang (14.0.6)

[warning] variable name 'os' is too short, expected at least 3 characters

(readability-identifier-length)